### ISSUE TITLE
sass --compass --watch ignores --load-path; problem with Compass sprites

### ISSUE THREAD

**Issue by mgol** [sass/sass#681]
I am using the technique described here:
http://compass-style.org/help/tutorials/spriting/
to automatically generate sprites from images. However, I can't just run `compass watch` from a command line because I need to generate files with source maps (only those ones work with current Chrome Canary and the same will be in Chrome stable in a couple of weeks). My configuration for Compass file, `config.rb`:

```
http_path = "/"
css_dir = "styles"
sass_dir = "styles"
images_dir = "img"
javascripts_dir = "js"
fonts_dir = "fonts"
output_style = :expanded
relative_assets = true
```

Since I can't just invoke `compass watch`, I do sth like that:

```
sass --compass --sourcemap --watch screen.scss:screen.css
```

It worked like a charm until I tried to create sprites. After putting an import:

```
@import "sprite-dir/*.png";
```

SASS complains:

```
Compass::SpriteException: No files were found in the load path matching "sprite-dir/*.png". Your current load paths are: ./images
```

Providing an option `--load-path ../img` doesn't change anything in this message. Thus, I created a symbolic link `images -> ../img`, but then in the output file I get:

``` css
background: url('/images/sprite-dir-SOME_HASH.png') no-repeat;
```

which is incorrect as `images` is not directly inside the root directory.

To summarize, I'd expect the `--load-path` option to work correctly here. I need at least to get one of the two following results (in preference order):

``` css
background: url('../img/sprite-dir-SOME_HASH.png') no-repeat;
background: url('images/sprite-dir-SOME_HASH.png') no-repeat;
```

I know no method to get it to work currently...

I use the latest SASS code from repository (and the current Compass stable gem if that matters here).

**Comment by nex3** [comment]
@chriseppstein this is more your purview than mine.

**Comment by chriseppstein** [comment]
It seems that the compass config file is not getting found because the
image directory path mentioned in the error does not match that in the
config. Can you please show me what your directory structure looks like,
what is your working directory and where is the config file.

**Comment by mgol** [comment]
Ah, I was using it from the `styles` directory instead of the `static` one. Anyway, why didn't the `--load-path` option work?

Anyway, my config is as follows:

```
project/
    static/
        config.rb
        img/
        js/
        styles/
```

**Comment by nex3** [comment]
`--load-path` tells Sass where to find Sass files to load; it's not meant to be used with images.

**Comment by chriseppstein** [comment]
@mzgol you need to set up the sprite load paths in your compass config:

``` ruby
sprite_load_path << "/path/to/sprite/root/folder"
```

**Comment by chriseppstein** [comment]
@mzgol let's move this over to the compass issue tracker.
